modules/seqexec/web/client/src/main/scala/seqexec/web/client/model/tabs.scala

```diff
@@ -81,24 +90,34 @@ object SeqexecTab {
     GenPrism[SeqexecTab, PreviewSequenceTab]
   val instrumentTab: Prism[SeqexecTab, InstrumentSequenceTab] =
     GenPrism[SeqexecTab, InstrumentSequenceTab]
+  val calibrationTab: Prism[SeqexecTab, CalibrationQueueTab] =
+    GenPrism[SeqexecTab, CalibrationQueueTab]
   val sequenceTab: Prism[SeqexecTab, SequenceTab] =
     Prism.partial[SeqexecTab, SequenceTab] {
       case p: PreviewSequenceTab    => p
       case i: InstrumentSequenceTab => i
     }(identity)
 }
 
-final case class CalibrationQueueTab(tableState: TableState[StepsTable.TableColumn], state: BatchExecState) extends SeqexecTab {
+@Lenses
+final case class CalibrationQueueTab(
+  tableState: TableState[StepsTable.TableColumn],
+  state:      BatchExecState,
+  observer:   Option[Observer])
```

Comment:
This should be on the core model, but lacking that we need to keep it on the client model, hence this field